crates/debugger/src/lib.rs

```diff
@@ -143,12 +156,23 @@ pub fn AppStatsView(hooks: &mut Hooks) -> Element {
             external_time.apply(sample.external_time);
         }
 
-        set_measurements((frame_time, external_time))
+        set_measurements((frame_time, external_time));
```

Comment:
So if I've gotten this correctly, it will use my previous `Measurement` to provide a moving average, min, and max from input processing to the end of the rendering system.

I don't seem to find a use of the `Display` impl for FrameTimings to report each frame, where is that, and is it just the latest or also a moving average?